### ISSUE TITLE
Do not generate violation in SuppressWarningsHolder for empty SuppressWarnings

### ISSUE THREAD

**Issue by mkordas** [checkstyle/checkstyle#1187]
Follow-up to #1158.

SuppressWarningsHolder generate violation in SuppressWarningsHolder for empty SuppressWarnings. As agreed, there is no point in raising such warning.

`SuppressWarnings` annotation cannot be empty. This is illegal code:
```java
@SuppressWarnings
volatile int[] e;
```

I wonder why FindBugs has such non-compilable test case: https://code.google.com/p/findbugs/source/browse/findbugsTestCases/src/java/VolatileUsage2.java.

**Comment by romani** [comment]
> I wonder why FindBugs has such non-compilable test case

 It is just smth different: ```import edu.umd.cs.findbugs.annotations.SuppressWarnings;```

**Comment by mkordas** [comment]
Apparently I was blind :) Looks like this null-check is needed then, but I'll remove logging warning anyway.

**Comment by romani** [comment]
oh yes, even I missed that.

checkstyle is not type aware so, we have to be aware that user could have its own implementation with similar name.  Any attempts to make a decision base on class implementation or methods signature is error prone.

**Comment by romani** [comment]
merged.
